### PR TITLE
Model, view, controller and  postgresql setup

### PR DESCRIPTION
**Pull request by mikolaslorant**
## Summary
-Model, view and controller added. 
-Added postgres configuration.

## Trello card
-MVC - Views
-MVC - Controllers
-MVC - Models

## Screenshot
screenshot from 2018-08-22 12-54-16

### CODE REVIEW

File: src/Models/Movie.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace MlNetCore.Models
+{
+    public class Movie
+    {
+        public int ID { get; set; }
```
Comment (gussiciliano) [inline src/Models/Movie.cs]: change this for`Id`

File: src/Models/Movie.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        public string Title { get; set; }
+        public DateTime ReleaseDate { get; set; }
+        public string Genre { get; set; }
+        public decimal Price { get; set; }
+    }
+}
```
Comment (gussiciliano) [inline src/Models/Movie.cs]: add eof

File: src/Views/HelloWorld/index.cshtml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+@{
+    ViewData["Title"] = "Movie List";
+}
+
+<h2>My Movie List</h2>
+
+<p>Hello from our View Template!</p>
```
Comment (gussiciliano) [inline src/Views/HelloWorld/index.cshtml]: add eof

File: src/Views/HelloWorld/index.cshtml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@{
+    ViewData["Title"] = "Movie List";
+}
+
+<h2>My Movie List</h2>
+
```
Comment (gussiciliano) [inline src/Views/HelloWorld/index.cshtml]: delete this empty line

File: src/Views/HelloWorld/Welcome.cshtml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@{
+    ViewData["Title"] = "Welcome";
+}
+
+<h2>Welcome</h2>
+
```
Comment (gussiciliano) [inline src/Views/HelloWorld/Welcome.cshtml]: delete this empty line

File: src/Views/HelloWorld/Welcome.cshtml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+<ul>
+    @for (int i = 0; i < (int)ViewData["NumTimes"]; i++)
+    {
+        <li>@ViewData["Message"]</li>
+    }
+</ul>
```
Comment (gussiciliano) [inline src/Views/HelloWorld/Welcome.cshtml]: add eof

File: src/Controllers/HelloWorldController.cs [hunk 7]
```diff
@@ -18,9 +18,11 @@ public IActionResult Index()
             return View();
         }
 
-        public string Welcome()
+        public IActionResult Welcome(string name, int numTimes = 1)
```
Comment (gussiciliano) [inline src/Controllers/HelloWorldController.cs]: remove the `int numTimes` :P

File: src/Views/HelloWorld/Welcome.cshtml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+@{
+    ViewData["Title"] = "Welcome";
+}
+
+<h2>Welcome</h2>
+
+<ul>
+    @for (int i = 0; i < (int)ViewData["NumTimes"]; i++)
```
Comment (gussiciliano) [inline src/Views/HelloWorld/Welcome.cshtml]: delete this for

**Review by gussiciliano — CHANGES_REQUESTED** [review]

**Comment by mikolaslorant** [comment]
Already corrected requested changes

**Review by gussiciliano — APPROVED** [review]

**Pull request merged**